R/save_output.R

```diff
@@ -32,6 +33,7 @@ save_meta <- function(df, labels, folder, file, title, contact, source, note="",
     skimr::skim(df) %>% kable()
     sink()
   }
+
```

Comment:
Here is the version of the save_meta() function I used to create the oic and housing_price readme and txt files (I found that it improved formatting). It caused a merge conflict when I submitted a pull, so I had it revert back to your version. Here is the code I had created that I had to remove:

save_meta <- function(df, labels, folder, file, title, contact, source, notes, apd = F) {

  # generate metadata
  sink(paste0(folder, "\\/", file, ".txt"), append = apd)
  cat("Title: ", title)
  cat("\nContact: ", contact)
  cat("\nSource: ", source)
  cat("\nLast updated: ", date(), "\n\n")

  print(labels %>% kable())
  cat("\n\n")

  skimr::skim_with(numeric = list(hist = NULL), integer = list(hist = NULL))
  skimr::skim(df) %>% kable()
  
  cat("\n\n")  
  cat("Notes: ", df_notes)
  
  sink()

  # create README
  sink(paste0(folder, "\\/README.md"), append = apd)

  cat("Title: ",title,"\ ")
  cat("\nContact: ", contact,"\ ")
  cat("\nSource: ", source,"\ ")
  cat("\nLast updated: ", date(), "\n\n")

  print(labels %>% kable())
  cat("\n\n")

  skimr::skim(df) %>% kable()
  
  cat("\n\n")  
  cat("Notes: ", df_notes)
  
  sink()